### ISSUE TITLE
Regarding the first stage training time

### ISSUE THREAD

**Issue by colorfulandcjy0806** [hustvl/MaskAdapter#2]
Hello, thank you very much for your work! I have a question, during the first stage training of MaskAdapter, I noticed that the maximum training step size is 295717. Does it take so long to train? Training on my two 4090 graphics cards takes nearly 24 hours 😭

**Comment by owl-10** [comment]
Thank you for your interest in our work! For the first stage of training, we’ve set it to 20 epochs. You can also adjust the batch size to fit your needs. We’ve observed that different models converge at different speeds during the Mask-Adapter training process. For instance, it takes over 10 epochs for models like fc-clip-large and maftp-large to converge, which is why we set the training to 20 epochs for better results. On the other hand, maftp-base converges in just a few epochs.

It's important to note that open-vocabulary segmentation generally requires longer training times. For example, fc-clip can take a lot of time during training. According to the fc-clip paper, it takes around 3.2 days with 8 V100 GPUs. We’re also exploring ways to design models that can reduce training time.

Let us know if you have any more questions!

**Comment by colorfulandcjy0806** [comment]
Thank you for your answer. My question has been resolved. Thank you!

**Comment by colorfulandcjy0806** [comment]
Thank you for your answer. My question has been resolved. Thank you!
